annotate

```diff
@@ -1,4 +1,4 @@
 while read line ;
 do echo "----------- $line" >> bench.perf
-perf annotate --stdio $line >> bench.perf ;
+perf annotate -M${ASM_FORMAT} --stdio $line >> bench.perf ;
```

Comment:
Sometime, for testing purpose I'll manually run `./annotate`. In order for it to keep working, it would be great if you could add a default value if `ASM_FORMAT` was not set.

Reply:
I just pushed a new version! Sorry for the delay! Let me know if this satisfies!